compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/ComposeScene.skiko.kt

```diff
@@ -86,20 +86,15 @@ interface ComposeScene {
     var layoutDirection: LayoutDirection
 
     /**
-     * Represents the scene bounds relatively to window in pixels.
-     *
-     * The position is used to convert local coordinates to window ones.
-     *
-     * The size is used to impose constraints on the content. If the size is undefined, it can be
-     * set to `null`. In such a case, the content will be laid out without any restrictions and
-     * the window size will be utilized to bounds verification.
-     *
-     * TODO split boundsInWindow and size https://youtrack.jetbrains.com/issue/COMPOSE-964
+     * The size (in pixels) is used to impose constraints on the content. If the size is undefined,
+     * it can be set to `null`. In such a case, the content will be laid out without any
+     * restrictions and the window size will be utilized to bounds verification.
      */
-    var boundsInWindow: IntRect?
+    var size: Size?
```

Comment:
```suggestion
    var size: IntSize?
```

`Int` instead of `Float` was chosen before because it was the main type of the layout subsystem, and the old `scene.constraints` / new `scene.size` are part of it. It is the main type because it should work with physical pixels to avoid anti aliasing during rendering lines of the layouted content.

Also, internally we don't need Float - it is either used for comparing, or converted to Int.

We can discuss online.

Reply:
Done